SMLHelper/Handlers/CustomFishHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace SMLHelper.V2.Handlers
+{
+    using System;
+    using System.Collections.Generic;
+    using SMLHelper.V2.FishFramework;
+    using SMLHelper.V2.Assets;
+
+    public static class CustomFishHandler
```

Comment:
As another standard practice for anything that is to be used as a public API, make sure to include XML documentation for all public classes, methods, fields, and properties.